web/tests/copy_and_paste.test.js

```diff
@@ -121,9 +121,13 @@ run_test("paste_handler_converter", () => {
     assert.equal(copy_and_paste.paste_handler_converter(input), "1. text");
 
     // Heading
+    // Changed the output format by removing the "#" symbol from the heading "Zulip overview".
+    // This ensures that the pasted heading no longer includes Markdown-style heading syntax (#),
+    // and is treated as normal text, improving consistency when pasting formatted content.
+
     input =
         '<meta http-equiv="content-type" content="text/html; charset=utf-8"><h1 style="box-sizing: border-box; font-size: 2em; margin-top: 0px !important; margin-right: 0px; margin-bottom: 16px; margin-left: 0px; font-weight: 600; line-height: 1.25; padding-bottom: 0.3em; border-bottom: 1px solid hsl(216, 14%, 93%); color: hsl(210, 12%, 16%); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-style: initial; text-decoration-color: initial;">Zulip overview</h1><p>normal text</p>';
-    assert.equal(copy_and_paste.paste_handler_converter(input), "# Zulip overview\n\nnormal text");
+    assert.equal(copy_and_paste.paste_handler_converter(input), "Zulip overview\n\nnormal text");
```

Comment:
@Mysooo we want to preserve the `#` in this case, but if there was no `meta` or `p` tag here, we want to drop the `#`. Please read the related issue and the CZO discussion linked there for further clarity.

Reply:
hi @amanagr,
I have implemented the necessary changes as discussed. Could you please review my pull request.